README.rst

```diff
@@ -67,36 +69,73 @@ Features
 Installation
 ============
 
-PennyLane-Forest requires both PennyLane and pyQuil. It can be installed via ``pip``:
+Installation of PennyLane-Forest, as well as all required Python packages mentioned above, can be installed via ``pip``:
+::
 
-.. code-block:: bash
+   	$ python -m pip install pennylane-forest
 
-    $ python -m pip install pennylane-forest
 
+Make sure you are using the Python 3 version of pip.
 
-.. installation-end-inclusion-marker-do-not-remove
+Alternatively, you can install PennyLane-Forest from the source code by navigating to the top directory and running
+::
+
+	$ python setup.py install
+
+Dependencies
+~~~~~~~~~~~~
+
+.. highlight:: bash
+
+PennyLane-Forest requires the following libraries be installed:
+
+* `Python <http://python.org/>`_ >=3.6
+
+as well as the following Python packages:
+
+* `PennyLane <http://pennylane.readthedocs.io/>`_
+* `pyQuil <http://docs.rigetti.com/>`_
```

Comment:
Might be better to link directly to the pyQuil docs:
![image](https://user-images.githubusercontent.com/6934626/98719489-d193ff00-235d-11eb-9886-5e57d73658ed.png)

```suggestion
* `pyQuil <https://pyquil-docs.rigetti.com/en/stable/>`_
```